### PR TITLE
Added the instance_type to report

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [ ] bug
- [ ] enhancement
- [x] documentation
- [ ] dependencies

## Description

Added the instance_type to the cloudsensei report

### CODE REVIEW

File: cloudsensei/lambda_function.py [hunk 1]
```diff
@@ -97,7 +98,7 @@ def organize_message_to_send_slack(self, resources_list: dict):
         """
 
         divider = {"type": "divider"}
-        keys_list = ['User', 'Region', 'Name', 'InstanceId', 'LaunchDate', 'RunningDays']
+        keys_list = ['User', 'Region', 'Name', 'InstanceId', 'LaunchDate', 'RunningDays', 'InstanceType']
```
Comment (ebattat) [inline cloudsensei/lambda_function.py:101]: can you locate InstanceType before InstanceId ?

File: cloudsensei/lambda_function.py [hunk 2]
```diff
@@ -121,7 +122,7 @@ def organize_message_to_seng_mail(self, resources_list: dict):
         :param resources_list:
         :return:
         """
-        keys_list = ['User', 'Region', 'Name', 'InstanceId', 'LaunchDate', 'State',  'RunningDays']
+        keys_list = ['User', 'Region', 'Name', 'InstanceId', 'LaunchDate', 'State',  'RunningDays', 'InstanceType']
```
Comment (ebattat) [inline cloudsensei/lambda_function.py:125]: can you locate InstanceType before InstanceId ?

File: cloudsensei/lambda_function.py [hunk 3]
```diff
@@ -77,7 +77,8 @@ def get_resources(self):
                             long_running_instances_by_user.setdefault(user.lower(), {}).setdefault(region_name, []).append(
                                 {'InstanceId': resource.get('InstanceId'),
                                  'Name': name, 'LaunchDate': str(launch_time),
-                                 'RunningDays': f"{days} days", 'State': resource.get('State', {}).get('Name')})
+                                 'RunningDays': f"{days} days", 'State': resource.get('State', {}).get('Name'),
+                                 'InstanceType': resource.get('InstanceType')})
```
Comment (ebattat) [inline cloudsensei/lambda_function.py:81]: can you change the order also here 'InstanceType' before 'RunningDays' ?

Comment (athiruma) [reply cloudsensei/lambda_function.py:81]: No need, It is the data storing/ returning variable.

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**